processing/src/main/java/io/druid/segment/IndexMerger.java

```diff
@@ -200,6 +200,9 @@ public File persist(
                 indexSpec.getBitmapSerdeFactory().getBitmapFactory()
             )
         ),
+        // if index is not rollup, then it should be not rollup here
+        // if index is rollup, then it is no need to rollup again
+        false,
```

Comment:
whether to rollup or not also affects Rowboat ordering, 
In case we want to do rollup, and two intermediate segments are persisted but with different row ordering (based on timestamps only and not considering dimensions),  
I wonder if this could this cause rows to be not rolled up during final merging of intermediate segments ?